### PR TITLE
tagged packets hits vlan port, check ARP

### PR DESCRIPTION
**Pull request by ganglyu**
Signed-off-by: Gang Lv ganglv@microsoft.com

### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911

#### What is the motivation for this PR?
Need new test: tagged packets hits vlan port, check ARP

#### How did you do it?
Send tagged GARP packet to each vlan port, check received packet on other vlan ports, and check ARP table.

#### How did you verify/test it?
Run this test case with t0 topology and t0-56-po2vlan topology.

### CODE REVIEW

File: tests/arp/test_tagged_arp.py [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+                vlan_port['permit_vlanid'].append(vlan['vlanid'])
+        if 'pvid' in vlan_port:
+            vlan_ports_list.append(vlan_port)
+
+    return vlan_ports_list
+
+
+def enable_arp(duthost, cfg_facts, enable):
+    vlan_members = cfg_facts.get('VLAN_MEMBER', {})
+    on_cmd = "echo 1 > /proc/sys/net/ipv4/conf/%s/arp_accept"
```
Comment (qiluo-msft) [inline tests/arp/test_tagged_arp.py:108]: >echo 1 > /proc/sys/net/ipv4/conf/%s/arp_accept [](http://example.com/codeflow?start=14&length=46)

Is it really necessary? What is the default behavior of sonic image?

File: tests/arp/test_tagged_arp.py [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+    Verify packets egress without tag from ports whose PVID same with ingress port.
+    Verify packets egress with tag from ports who include VLAN ID but PVID different from ingress port.
+    verify show arp command on DUT.
+    """
+    duthost = duthosts[rand_one_dut_hostname]
+    for vlan_port in vlan_ports_list:
+        port_index = vlan_port["port_index"][0]
+        # Send GARP packets to switch to populate the arp table with dummy MACs for each port
+        # Totally 10 dummy MACs for each port, send 1 packet for each dummy MAC
+        dummy_macs = ['{}:{:02x}:{:02x}'.format(DUMMY_MAC_PREFIX, port_index, i+1)
```
Comment (qiluo-msft) [inline tests/arp/test_tagged_arp.py:214]: >port_index [](http://example.com/codeflow?start=66&length=10)

We have some SKU with so many ports. Will that be a problem if we want to test arp on those topo?

File: tests/arp/test_tagged_arp.py (hunk 1 above)
Comment (ganglyu) [reply tests/arp/test_tagged_arp.py:34]: Test again, still need to enable arp_accept.

Comment (prsunny) [reply tests/arp/test_tagged_arp.py:34]: arp_accept is for garp. Why don't the test send regular arp request packet here?

Comment (ganglyu) [reply tests/arp/test_tagged_arp.py:34]: > arp_accept is for garp. Why don't the test send regular arp request packet here?

I have verified arp request packet, but still got the same result.

**Pull request merged**